.gitlab-ci.yml

```diff
@@ -1,15 +1,18 @@
 # The testing matrix considers ruby/puppet versions supported by SIMP and PE:
 #
-# https://puppet.com/docs/pe/2017.3/overview/component_versions_in_recent_pe_releases.html
+# https://puppet.com/docs/pe/latest/overview/component_versions_in_recent_pe_releases.html
```

Comment:
A link to the documentation under `/latest/` will not necessarily contain this data in the future. 

Data about historical releases is periodically removed from newer versions of the documentation, particularly on the "component releases" page (note that the versions start with 2016.4, the oldest LTS).  Documentation URLs are also occasionally restructured from version to version.  

We've been linking to a specific version of the documentation to ensure it has the correct information. 

```suggestion
# https://puppet.com/docs/pe/2018.1/component_versions_in_recent_pe_releases.html
```